app/utils/exception_handlers.py

```diff
@@ -0,0 +1,21 @@
+from fastapi.responses import JSONResponse
+from fastapi import Request
+from app.exceptions.GroupNotFoundException import GroupNotFoundException
+from app.exceptions.UserNotFoundException import UserNotFoundException
+
+
+async def user_not_found_exception_handler(
```

Comment:
Maybe move this in the `exceptions` module/package

Reply:
Done!